src/js/analyses/components/Create/Quick.js

```diff
@@ -34,6 +46,22 @@ const QuickAnalyzeSelected = styled.span`
     margin: 0 15px 0 auto;
 `;
 
+const StyledError = styled.div`
+    margin -5px 0px 10px
+`;
```

Comment:
`QuickAnalyzeError`.

Only use the `Styled` prefix when you are using it as the top-level element in your component (eg. `StyledQuickAnalyze`).

The preferred solution here is actually to use:
```js
const QuickAnalyzeError = Styled(InputError)``;
```
And redefine its margin accordingly. You are adding an extra `div` here.

Reply:
Ahh right, good point. Changed.